easybuild/easyconfigs/s/statsmodels/statsmodels-0.6.1-foss-2017a-Python-2.7.13.eb

```diff
@@ -0,0 +1,38 @@
+easyblock = 'Bundle'
+
+name = 'statsmodels'
+version = '0.6.1'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'http://statsmodels.sourceforge.net/'
+description = """Statsmodels is a Python module that allows users to explore data, estimate statistical models,
+and perform statistical tests."""
+
+toolchain = {'name': 'foss', 'version': '2017a'}
+
+# this is a bundle of Python packages
+exts_defaultclass = 'PythonPackage'
+
+dependencies = [('Python', '2.7.13')]
+
+exts_list = [
+    ('patsy', '0.4.1', {
+        'source_tmpl': 'patsy-%(version)s.zip',
+        'source_urls': ['https://pypi.python.org/packages/source/p/patsy'],
+        'checksums': ['dc1cc280045b0e6e50c04706fd1e26d2a00ea400aa112f88e8142f88b0b7d3d4'],
+    }),
+    (name, version, {
+        'source_tmpl': 'statsmodels-%(version)s.zip',
+        'source_urls': ['https://pypi.python.org/packages/source/statsmodels'],
+        'checksums': ['4313dd4cbf1ebadef3661f6efd895132b615016e9bee560b5fdfd80c74dd422f'],
+    }),
+]
+
```

Comment:
@timeu One of the existing `statsmodels` easyconfigs has this, should be included here too:

```
# specify that Bundle easyblock should run a full sanity check, rather than just trying to load the module
full_sanity_check = True
```